app/controllers/user_filters_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
     active_filter_options = params.require(:activeFilterOptions).collect do |filter|
       {
         filterOption: filter.require(:filterOption).permit(:name, :title, :description, :type, :hasTimestamp, :allowRange, options: []),
-        value: filter.permit(:value)[:value] || filter.require(:value) || false,
+        value: permitted_filter_params[:value] || filter.require(:value) || false,
```

Comment:
Where is `permitted_filter_params` - think this shouldn't have changed.

Reply:
whoops copy pasted - should be all set now.  also squashed